docker-build.sh

```diff
@@ -135,19 +135,54 @@ buildOpenJDKViaDocker()
      dockerEntrypoint=(--entrypoint "/bin/sh" "${BUILD_CONFIG[CONTAINER_NAME]}" -c "echo 'DEBUG DOCKER BUILD\\nTo build jdk run\\n/openjdk/sbin/build.sh'; /bin/bash")
   fi
 
-  # shellcheck disable=SC2140
-  # Pass in the last important variables into the Docker container and call
-  # the /openjdk/sbin/build.sh script inside
-  ${BUILD_CONFIG[DOCKER]} run \
+  #if both dockerMode and gitSshAccess are unbounded
+  if [ ! -n "${dockerMode[@]:-}" ]  && [ ! -n "${gitSshAccess[@]:-}" ]; then
```

Comment:
Seems like a fair amount of Don't Repeat Yourself (DRY) here - is it possible to extract common logic?

Reply:
Done. I've changed where the debug error message executes in the script, but it looks the same as It did before. This is because I was having a lot of issues with missing speech marks when adding `dockerMode` and `gitSshAccess` onto `commandString`